tests/Fixtures/TestBundle/Entity/UserResource.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Post;
+use ApiPlatform\Tests\Fixtures\TestBundle\Dto\UserResetPasswordDto;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ApiResource()]
```

Comment:
Same here, use annotations for PHP 7 compat.

Reply:
I'm working on `main` branch. The `ApiResource` annotation doesn't exist anymore (nor the whole `src/Annotation` directory).
Should I continue on `main` branch with attributes ?

Reply:
I mean, even if they have been renamed, they can still be used as Doctrine annotations instead of as PHP 8 attributes isn't it?

Using annotations will allow to run the test suite with PHP 7, which is still supported by API Platform 2.7.

Reply:
We don't test on 2.7 we test on 3.x with a Processor.